openapi/components/schemas/GatewayAccountConfig/Ezeebill.yaml

```
@@ -0,0 +1,42 @@
+description: Ezeebill gateway configuration.
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        description: Ezeebill credentials object.
+        properties:
+          merchantId:
+            type: string
+            description: MID in the Ezeebill.
```

nitpick: I think we should try to use complete sentences
```suggestion
            description: Merchant ID (MID) of the Ezeebill gateway.
```